### PR TITLE
EDX-2653 Update XRT models for v2

### PR DESCRIPTION
**Pull request by weichou1229**
- Update XRT models for v2
- Add colon char for XRT resource name

Signed-off-by: bruce <weichou1229@gmail.com>

### CODE REVIEW

File: xrtmodels/response.go [hunk 1]
```diff
@@ -2,33 +2,51 @@
...
+}
+
+func (result BaseResult) Error() errors.EdgeX {
+	switch result.Status {
+	case XrtSdkStatusOk:
+		return nil
+	case XrtSdkStatusNotFound:
+		return errors.NewCommonEdgeX(errors.KindEntityDoesNotExist, result.ErrorMessage, nil)
+	case XrtSdkStatusNotSupported:
+		return errors.NewCommonEdgeX(errors.KindServiceUnavailable, result.ErrorMessage, nil)
```
Comment (cloudxxx8) [inline xrtmodels/response.go:42]: should it be `KindNotImplemented`?

Comment (cloudxxx8) [inline xrtmodels/response.go:42]: should it be `KindNotImplemented`?

Comment (weichou1229) [reply xrtmodels/response.go:42]: The XRT will throw the `DEVSDK_STATUS_NOT_SUPPORTED` status code when `discovery not supported`, so I think it's fine to use `KindNotImplemented`.
https://github.com/IOTechSystems/xrt/blob/v2.0-branch/src/c/devsdk/operations.c

Comment (cloudxxx8) [reply xrtmodels/response.go:42]: In this case, KindNotImplemented fit the situation

Comment (weichou1229) [reply xrtmodels/response.go]: Got it, modified.

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**